x-pack/plugins/secops/public/components/timeline/body/renderers/zeek_signature.tsx

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiBadge, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { get } from 'lodash/fp';
+import * as React from 'react';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+
+import { Ecs } from '../../../../graphql/types';
+import { DragEffects, DraggableWrapper } from '../../../drag_and_drop/draggable_wrapper';
+import { escapeDataProviderId } from '../../../drag_and_drop/helpers';
+import { GoogleLink, TotalVirusLink } from '../../../links';
+import { Provider } from '../../../timeline/data_providers/provider';
+
+import * as i18n from './translations';
+
+const Badge = styled(EuiBadge)`
+  vertical-align: top;
+`;
+
+const TokensFlexItem = styled(EuiFlexItem)`
+  margin-left: 3px;
+`;
+
+const LinkFlexItem = styled(EuiFlexItem)`
+  margin-left: 6px;
+`;
+
+type StringRenderer = (value: string) => string;
+
+export const defaultStringRenderer: StringRenderer = (value: string) => value;
+
+export const moduleStringRenderer: StringRenderer = (value: string) => {
+  const split = value.split('.');
+  if (split.length >= 2 && split[1] != null) {
+    if (split[1] !== '') {
+      return split[1];
+    } else {
+      return split[0];
+    }
+  } else {
+    return value;
+  }
+};
+
+export const droppedStringRenderer: StringRenderer = (value: string) => `Dropped:${value}`;
+
+export const md5StringRenderer: StringRenderer = (value: string) => `md5: ${value.substr(0, 7)}...`;
+
+export const sha1StringRenderer: StringRenderer = (value: string) =>
+  `sha1: ${value.substr(0, 7)}...`;
+
+export const DraggableZeekElement = pure(
+  ({
+    id,
+    field,
+    value,
+    stringRenderer = defaultStringRenderer,
+  }: {
+    id: string;
+    field: string;
+    value: string | null;
+    stringRenderer?: StringRenderer;
+  }) =>
+    value != null ? (
+      <TokensFlexItem grow={false}>
+        <DraggableWrapper
+          dataProvider={{
+            and: [],
+            enabled: true,
+            id: escapeDataProviderId(`zeek-${id}-${field}-${value}`),
+            name: value,
+            excluded: false,
+            kqlQuery: '',
+            queryMatch: {
+              field,
+              value,
```

Comment:
consider using `escapeQueryValue` here

Reply:
Did but wrote this up as there is still a bug with how it escapes `::`
https://github.com/elastic/ingest-dev/issues/315